src/transformState.js

```diff
@@ -5,7 +5,26 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    switch (actions[i].type) {
+      case 'addProperties':
+        Object.assign(state, actions[i].extraData);
+        break;
+
+      case 'removeProperties':
+        for (const char of actions[i].keysToRemove) {
+          delete state[char];
+        }
+        break;
+
+      default:
+        for (const key in state) {
+          delete state[key];
+        }
```

Comment:
it isn't default case it is case for 'clear'

use default for error handling, like
throw new Error('error description');

Reply:
fix please this comment